python/llm/dev/benchmark/all-in-one/config.yaml

```diff
@@ -14,4 +14,6 @@ test_api:
   - "optimize_model"
   - "pytorch_autocast_bf16"
   # - "transformer_int4_gpu"  # on arc
-  # - "optimize_model_gpu"  # on arc
\ No newline at end of file
+  # - "optimize_model_gpu"  # on arc
+  # - "original_model_cpu"
+  # - "original_model_gpu"
```

Comment:
add comments to describe different options

Reply:
Got it! I've added comments in config.yaml.